collectors/ebpf.plugin/multi_metadata.yaml

```diff
@@ -4,21 +4,21 @@ modules:
       plugin_name: ebpf.plugin
       module_name: filedescriptor
       monitored_instance:
-        name: ebpf filedescriptor
+        name: Filedescriptor
```

Comment:
We probably still want `eBPF` in the name for most of these, so that it’s clear from looking at the cards that the data is collected using eBPF.

Reply:
I had a talk with @Ancairon about this hours ago, this was the motive I modified, but I am returning now.